### DISCUSSION TITLE
Undefined variable $linkGroups

### DISCUSSION THREAD

**Discussion by feelgoodvince** [area17/twill#2465] [Q&A]
## Description
I'm updating to Twill 3.0 and I get the following error when I want to enter the admin panel.

### Steps to reproduce
Rename view folder & routes/admin.php
composer update to twill 3
php ./vendor/area17/twill/upgrade.php
php artisan migrate
php artisan twill:build

### Expected result

I expect to be able to login into the admin panel

### Actual result

Undefined variable $linkGroups (View: /Users/xxxxxx/Documents/xxxxxxx/xxxxxxxx/website/vendor/area17/twill/views/partials/navigation/_overlay_navigation.blade.php)

### Versions
Twill 3.0.2
PHP 8.2.3
Laravel 9.52.15
Mysql 5.7

**Comment by ifox** [comment]
Hi @feelgoodvince sorry about the delay... Were you able to get it sorted?

**Reply by feelgoodvince** [reply]
Hi @ifox , yes it's solved in the meantime

**Reply by jonrobinson** [reply]
@feelgoodvince finally upgrading my twill to 3.0 and running into the same issue. What was your solution?
